x-pack/plugins/infra/public/containers/logs/flyout_item.gql_query.ts

```diff
@@ -13,6 +13,10 @@ export const flyoutItemQuery = gql`
       logItem(id: $itemId) {
         id
         index
+        key {
+          time
```

Comment:
nitpick: we could DRY this up using the common `InfraTimeKeyFields` fragment in case the definition of a key changes :innocent: 

```typescript
import { sharedFragments } from '../../../common/graphql/shared'
```

and

```graphql
  query FlyoutItemQuery($sourceId: ID!, $itemId: ID!) {
    source(id: $sourceId) {
      id
      logItem(id: $itemId) {
        id
        index
        key {
          ...InfraTimeKeyFields
        }
        fields {
          field
          value
        }
      }
    }
  }

  ${sharedFragments.InfraTimeKey}
```